File: izinga-commons/src/main/kotlin/io/curiousoft/izinga/commons/order/events/NewOrderEvent.java
package io.curiousoft.izinga.commons.order.events;

import io.curiousoft.izinga.commons.model.Order;
import io.curiousoft.izinga.commons.model.StoreProfile;

public class NewOrderEvent extends OrderEvent {

    public NewOrderEvent(Object source, Order newOrder, String messengerId, StoreProfile receivingStore) {
        super(source, newOrder, messengerId, receivingStore);
    }
}

File: izinga-commons/src/main/kotlin/io/curiousoft/izinga/commons/order/events/NewOrderEvent.kt
package io.curiousoft.izinga.commons.order.events

import io.curiousoft.izinga.commons.model.Order
import io.curiousoft.izinga.commons.model.StoreProfile

class NewOrderEvent(source: Any, newOrder: Order, messengerId: String, receivingStore: StoreProfile) :
    OrderEvent(source, newOrder, messengerId, receivingStore)